src/components/site/linearProgress/LinearProgressMarker.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const LinearProgressMarker = ({ progress, completed, hoverable, children }) => (
```

Comment:
re-usable linear progress marker that is used to display the progress percentage, checkpoints, and finish point.